Dockerfile

```diff
@@ -53,7 +53,9 @@ RUN cd /tmp \
 # install libseccomp
 ARG LIBSECCOMP_VERSION
```

Comment:
It's a bit unfortunate that both the Dockerfile and script/release.sh now contain the version to use (was wondering if we had to persist this ARG as an ENV (`ENV LIBSECCOMP_VERSION=$LIBSECCOMP_VERSION`), and update the script to take that into account as a way to override the version? 🤔 


(Perhaps same for the source path), e.g. something like (haven't checked if I have the syntax correct)

```bash
local libseccomp_version="${LIBSECCOMP_VERSION-2.5.2)}"
local seccompdir ="${LIBSECCOMP_SRC-$(mktemp -d)}"
```